easybuild/easyconfigs/s/SUNDIALS/SUNDIALS-5.7.0-foss-2020b.eb

```diff
@@ -10,9 +10,9 @@ description = "SUNDIALS: SUite of Nonlinear and DIfferential/ALgebraic Equation
 toolchain = {'name': 'foss', 'version': '2020b'}
 toolchainopts = {'openmp': True, 'usempi': True, 'pic': True}
 
-source_urls = ['https://computation.llnl.gov/projects/sundials/download/']
+source_urls = ['https://github.com/LLNL/sundials/releases/download/v%(version)s/']
 sources = [SOURCELOWER_TAR_GZ]
-checksums = ['8d6dd094feccbb8d6ecc41340ec16a65fabac82ed4415023f6d7c1c2390ea2f3']
+checksums = ['48da7baa8152ddb22aed1b02d82d1dbb4fbfea22acf67634011aa0303a100a43']
```

Comment:
Huh, that's clearly wrong, not sure how this slipped in... Fixing with a force push.

Reply:
Actually, no, it's the correct checksum when downloading from GitHub, so the source tarball is not identical to the original one... 🤦